api/http/handler/edgestacks/edgestack_create.go

```diff
@@ -15,8 +16,12 @@ import (
 func (handler *Handler) edgeStackCreate(w http.ResponseWriter, r *http.Request) *httperror.HandlerError {
 	method, err := request.RetrieveRouteVariableValue(r, "method")
 	if err != nil {
-		return httperror.BadRequest("Invalid query parameter: method", err)
+		method, err = request.RetrieveQueryParameter(r, "method", true)
+		if err != nil {
+			return httperror.BadRequest("Invalid query parameter: method", err)
+		}
```

Comment:
This is not needed anymore because the handler will always be served a route with `{method}` path param